docs/modules/administrator-guides/pages/update.adoc

```diff
@@ -106,6 +106,9 @@ update notes below which apply to your version change.
 
 ==== Setup secure route checking
 
+This secure route checking code has been removed in version 2.1.x as this is an
```

Comment:
This should not be removed in 2. Only on master, which next release will be 3.

Reply:
Should the note say that it has been removed on master and will be removed on the next release or should we wait until release time and then have the commit ready.